sdk/core/azure-core-test/src/main/java/com/azure/core/test/models/MockTokenCredential.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.test.models;
```

Comment:
This doesn't feel like the right spot in the package to me - it's not really a model type. Perhaps `utils`?